sbom/examples/product/create.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python
+import argparse
+import json
+
+
+def create_sbom(cpeid, name, versionInfo, component_sbom):
+    # Collect component purls
+    packages = [
+        {
+            "SPDXID": "SPDXRef-product",
+            "name": name,
+            "versionInfo": versionInfo,
+            "supplier": "Organization: Red Hat",
+            "downloadLocation": "NOASSERTION",
+            "packageFileName": "",
+            "licenseConcluded": "",
```

Comment:
Not filled in yet, but these would have values.

Reply:
Do products themselves have different licenses that aren't just a compilation of all of the licenses of the individual components that are included in them?

Reply:
Oh, I don't think so. So maybe we'd leave out `licenseConcluded`.